### ISSUE TITLE
Update tackle to prompt user for a `zone` if a default is not set.

### ISSUE THREAD

**Issue by clarketm** [kubernetes/test-infra#13414]
**What would you like to be added**:
A compute **zone** selection prompt in`tackle` (*defaulting* to the `compute/zone` config value).

**Why is this needed**:
When using `tackle` to setup a personal prow cluster for the first time, if a default `zone` is not specified then the tool will throw a fatal error terminating the interactive session. 

```text
ERROR: (gcloud.container.clusters.create) One of [--zone, --region] must be supplied: Please specify location.
FATA[0030] Failed to select context                      error="create context: create cluster in clarketm-test: create cluster: exit status 1"
```

It would be intuitive (and useful) to prompt the user to input a zone (*defaulting* to the `compute/zone` config value if set for convenience) rather than throwing an arcane error at runtime.

**Comment by clarketm** [comment]
#13417

**Comment by clarketm** [comment]
closes #10687
